pkg/provisioners/heartbeats/config/heartbeat.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: eventing.knative.dev/v1alpha1
+kind: ClusterProvisioner
```

Comment:
how does this relate to `kind: Provisioner` in the new spec ?  I assume this is related to the current `300-clusterprovisioner.yaml` file ?

Reply:
Solved: the spec will have `kind: ClusterProvisioner` (see https://github.com/n3wscott/eventing/pull/8)